en/docs/guides/identity-verification/identity-verification-overview.md

```diff
@@ -0,0 +1,28 @@
+# Identity Verification
+
+Identity verification(IDV), is a crucial component of managing user access and verifying identities in systems and platforms. 
+It enables organizations to confirm the identity of users through various authentication methods, such as by 
+checking legal documents, like passports, national identification cards, and driver's licenses.
+
+IDV is a critical aspect of security and risk management, as it helps to prevent identity fraud and ensures that 
+only authorized individuals have access to sensitive systems and data. By having IDV, organizations can protect 
+themselves from potential threats and enhance the overall security of their systems.
+
+## Managing Identity Verification
+
+The user identity verification data are handled in WSO2 Identity Server and identity verification is primarily handled by the WSO2 Identity Server. 
```

Comment:
Shall we clarify this sentence a bit more?

Reply:
are both identity verification and identity verification data handled by WSO2 IS?

Reply:
Identity verification is performed by IS by connecting to 3rd party identity verification connectors through the available IDV IS connectors in the connector store. 

Identity verification status and some metadata related to the verification for attributes are managed in IS.

I have updated the description